src/main/java/org/folio/rest/impl/TenantRefAPI.java

```diff
@@ -104,17 +110,16 @@ public void postTenant(TenantAttributes ta, Map<String, String> headers,
       }
       TenantLoading tl = new TenantLoading();
 
-      tl.withKey(REFERENCE_KEY).withLead(REFERENCE_LEAD);
       tl.withIdContent();
+
+      tl.withKey(REFERENCE_KEY).withLead(REFERENCE_LEAD);
       for (String p : refPaths) {
         tl.add(p);
       }
       tl.withKey(SAMPLE_KEY).withLead(SAMPLE_LEAD);
-      tl.add("instances", "instance-storage/instances");
-      tl.withIdContent();
-      tl.add("holdingsrecords", "holdings-storage/holdings");
-      tl.add("items", "item-storage/items");
-      tl.add("instance-relationships", "instance-storage/instance-relationships");
+      for (String p : samplePaths) {
+        tl.add(p);
+      }
```

Comment:
The previous code appears to create an explicit mapping between the type of record and the path.

How does the new code do this, as it appears to only add the path?